exercises/sum.cpp

```diff
@@ -1,8 +1,18 @@
-/*
-  Write a program that takes as input two numbers and print the sum.
-
-  Output:
-  Insert the first number: 1
-  Insert the second number: 2
-  Sum: 3
-*/
+#include <iostream>
+using namespace std;
+
+int main() {
+    double numero1, numero2, somma;
+
+    cout << "Insert the first number: ";
+    cin >> numero1;
+
+    cout << "Insert the second number: ";
+    cin >> numero2;
+
+    somma = numero1 + numero2;
+
+    cout << "La somma dei due numeri è: " << somma << endl;
+
+    return 0;
```

Comment:
Potresti definire un `#define EXIT_SUCCESS 0;` e usarlo al posto dello 0 per dare più significatività al codice che viene letto:

```suggestion
    return EXIT_SUCCESS;
```

Allo stesso modo puoi definire un `#define EXIT_FAILURE 1` da poter usare in caso di errore.